bulk/sensor_vocab.csv

```diff
@@ -69,6 +69,7 @@ CTD Pumped: CTDBP Series O,Sea-Bird Electronics,SBE 16plus V2
 CTD Pumped: CTDBP Series P,Sea-Bird Electronics,SBE 16plus-IM V2
 Diffuse Vent Fluid 3-D Temperature Array: TMPSF Series A,RBR Global,XR-420-T24
 Direct Covariance Flux: FDCHP Series A,WHOI,DCFS
```

Comment:
Should change this from:
DCFS
to:
DCFS with Windmaster Pro anemometer.

Reply:
updated

Reply:
and changed back